dev/ci/ci-compcert.sh

```diff
@@ -9,7 +9,11 @@ git_download compcert
 
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
-export COQCOPTS='-native-compiler no -w -undeclared-scope -w -omega-is-deprecated'
+export COQCOPTS='-w -undeclared-scope -w -omega-is-deprecated'
```

Comment:
Doesn't work
>ocamlopt.opt got signal and exited
Error: Native compiler exited with status 2

Reply:
Indeed, compiling compcert with native-compiler on can work but the memory requirements don't seem reasonnable for CI, c.f.: https://coq.zulipchat.com/#narrow/stream/250632-Coq-Platform-devs-.26-users/topic/coq-native/near/390873410
Let's forget about it for now.